etcdctlv3/command/txn_command.go

```diff
@@ -18,21 +18,31 @@ import (
 	"bufio"
 	"fmt"
 	"os"
+	"regexp"
 	"strconv"
 	"strings"
 
 	"github.com/coreos/etcd/Godeps/_workspace/src/github.com/spf13/cobra"
 	"github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context"
+	"github.com/coreos/etcd/clientv3"
 	pb "github.com/coreos/etcd/etcdserver/etcdserverpb"
 )
 
+var (
+	txnInteractive bool
+	txnHex         bool
+)
+
 // NewTxnCommand returns the cobra command for "txn".
 func NewTxnCommand() *cobra.Command {
-	return &cobra.Command{
-		Use:   "txn",
+	cmd := &cobra.Command{
+		Use:   "txn [options]",
 		Short: "Txn processes all the requests in one transaction.",
 		Run:   txnCommandFunc,
 	}
+	cmd.Flags().BoolVar(&txnInteractive, "i", false, "input transaction in interactive mode")
+	cmd.Flags().BoolVar(&txnHex, "i", false, "input transaction in interactive mode")
```

Comment:
cmd.Flags().BoolVar(&txnHex, "hex", false, "accept and print out key-value as hex encode string")